### PR TITLE
Add example for global polyfill to README.md

### PR DESCRIPTION
**Pull request by andrew-pyle**
Based on discussions in Issue #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,17 @@ const serialized = serialize({any: 'serializable'});
 const deserialized = deserialize(serialized);
 ```
 
+#### Global Polyfill
+```js
+// Attach the polyfill as a Global function
```
Comment (WebReflection) [inline README.md:41]: please add *only if needed* to be extra clear.

**Comment by WebReflection** [comment]
I am OK with this MR but I don't understand why people can't just use `import structuredClone from '@poly'` instead of monkey patching the global ... I don't want to encourage global monkey patching, I'd love for developers to understand all they need is an import and use that in their mode *as if it is already the global one*.

This will backfire regardless others thinking they should patch the global ... let's see if that'd be the case.

**Comment by andrew-pyle** [comment]
@WebReflection Of course you are correct—people could just import the polyfill and use the imported module directly, like `import structuredClone from "@ungap/structured-clone"`. There would be all the regular benefits to that, encapsulation, unambiguous definition, etc.

But I think the reason people want to monkey patch the Global is simple—they would rather use the API defined in the Spec: `structuredClone()` as a global function.

I know that I usually just look on MDN and expect a polyfill to conform to the spec's API. It's difficult to infer the correct usage for every different polyfill, with CJS vs ESM, global vs import, NPM vs <script>. It's just so much easier to default to what's on [MDN](https://developer.mozilla.org/en-US/docs/Web/API/structuredClone). And in this case [MDN](https://developer.mozilla.org/en-US/docs/Web/API/structuredClone) says "structuredClone() global function".

**Comment by WebReflection** [comment]
> hey would rather use the API defined in the Spec: structuredClone() as a global function.

if you `import structuredClone from '@ungap/structured-clone'` and the global function exists, you will use **exactly that global function** without needing to patch the global at all ... this is maybe not clear in the README.

**Comment by andrew-pyle** [comment]
@WebReflection I do think a description of how this module falls back to the native implementation of `structuredClone()` would help convince users to avoid adding the polyfill to the global scope. Could you add that to the README?

I noticed the "fallback to native" feature while troubleshooting the infinite loop problem, but I would have never noticed without reading the code.

At the end of the day, users may just be thinking "polyfills go in global scope, done."

**Comment by WebReflection** [comment]
Some polyfill does (see custom elements or others) this one is special because it exposes internals too but in general if you import an utility you just use that utility and it’s not your duty to patch the global … either the polyfill does that or use the utility

**Pull request merged**